src/wallet/wallet.cpp

```diff
@@ -4387,7 +4387,7 @@ std::vector<COutput>::iterator OutputGroup::discard(const COutput& output) {
 std::vector<CInputCoin> OutputGroup::input_coins() const {
     std::vector<CInputCoin> coins;
     for (const auto& output : m_outputs) {
-        coins.emplace_back(output.tx, output.i);
+        coins.emplace_back(output.tx->tx, output.i);
```

Comment:
Shouldn't this change be part of 27872fa1cf4f7ca5e0c136554078448dc7f92d1d?

Reply:
Yes! Thanks for noticing.